cves/CVE-2013-2277.yml

```diff
@@ -90,7 +100,7 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: bdeb61ccc67911cfc5e20c7cfb1312d0501ca90a
-  note: ''
+  note: 'Fix in the most recent version.'
```

Comment:
Could you elaborate?

Reply:
I meant to say the commit was the most recent version published in CVE security.